libminifi/test/Utils.h

```diff
@@ -29,4 +30,11 @@
     return std::forward<T>(instance).method(std::forward<Args>(args)...); \
   }
 
-#endif  // LIBMINIFI_TEST_UTILS_H_
+std::string repeat(const std::string& str, std::size_t count) {
```

Comment:
This belongs to stringultis imho, but also a special case of join.
```
return join("", std::vector<std::string>(count, str));
```

Reply:
oh nice, I was looking in the StringUtils for something like this, and even found join, but did not put the pieces together

Reply:
reimplemented repeat using join